sycl/include/CL/sycl/detail/helpers.hpp

```diff
@@ -23,7 +23,7 @@ template <int dimensions, bool with_offset> struct item;
 template <int dimensions> class group;
 template <int dimensions> class range;
 template <int dimensions> struct id;
-template <int dimensions> class nd_item;
+template <int dimensions> struct nd_item;
```

Comment:
According to the SYCL specification nd_item should be class.
https://github.com/KhronosGroup/SYCL-Docs/blob/4a2dff3a779ae6fa4a477b0f02002c6ab82a3088/latex/headers/nditem.h

Reply:
As well as `id`.

Reply:
In public specification id still is a struct([sycl-1.2.1](https://www.khronos.org/registry/SYCL/specs/sycl-1.2.1.pdf))

Reply:
Pdf file is a bit outdated. Revision 5 doesn't include https://github.com/KhronosGroup/SYCL-Docs/pull/5/commits/51e006c4705b0397c95b8737484a650059f93cac fix.

Reply:
Then please do the alignment in a separate commit with reference to the source.

Reply:
Ok, ok..., I just cancel the fix in this line in this patch.
It should be a separate trivial but big fix changing struct to class for the classes: group, h_item, id, item, nd_range, nd_item.